entity-service-impl/src/main/java/org/hypertrace/entity/query/service/EntityQueryServiceImpl.java

```diff
@@ -609,7 +609,7 @@ private ResultSetMetadata buildMetadataForSelections(List<Expression> selections
     for (final Expression selection : selections) {
       final String columnName;
 
-      if (selection.hasAggregation()) {
+      if (selection.hasFunction()) {
```

Comment:
@skjindal93 This was missed in the previous PR and is required for aggregations to work.